nextflow.config

```diff
@@ -66,7 +66,7 @@ params {
   no_strelka_bp = null // Strelka will use Manta candidateSmallIndels if available
   pon = false // No default PON (Panel of Normals) file for GATK Mutect2 / Sentieon TNscope
   pon_index = false // No default PON index for GATK Mutect2 / Sentieon TNscope
-  ignore_soft_clipped_bases = null // GATK Mutect2
+  ignore_soft_clipped_bases = null // enable --dont-use-soft-clipped-bases for GATK Mutect2
```

Comment:
Is the function already used in the mutect2 step or just defined here?

Reply:
in used since we merged #230 actually, just forgot to add some docs